### PR TITLE
More base release 1.6 stuff - Get pipeline working again

### PR DESCRIPTION
**Pull request by ericvn**

### CODE REVIEW

File: content/en/news/releases/1.0.x/announcing-1.0/index.md [hunk 1]
```diff
@@ -121,7 +121,7 @@ be configured using [authentication policies](/docs/concepts/security/#authentic
 
 ### `istioctl`
 
-- Added the [`istioctl authn tls-check`](/docs/reference/commands/istioctl/#istioctl-authn-tls-check) command.
+- Added the [`istioctl authn tls-check`](/docs/reference/commands/istioctl) command.
```
Comment (frankbu) [inline content/en/news/releases/1.0.x/announcing-1.0/index.md]:
```suggestion
- Added the [`istioctl authn tls-check`](https://archive.istio.io/v1.0/docs/reference/commands/istioctl/#istioctl-authn-tls-check) command.
```

Comment (frankbu) [reply content/en/news/releases/1.0.x/announcing-1.0/index.md]: @ericvn Use this same pattern for all broken links in release notes.

File: content/en/docs/ops/common-problems/network-issues/index.md [hunk 2]
```diff
@@ -81,7 +81,7 @@ Otherwise, the mode defaults to `DISABLE` causing client proxy sidecars to make
 instead of TLS encrypted requests. Thus, the requests conflict with the server proxy because the server proxy expects
 encrypted requests.
 
-To confirm there is a conflict, check whether the `STATUS` field in the output of the [`istioctl authn tls-check`](/docs/reference/commands/istioctl/#istioctl-authn-tls-check) command
+To confirm there is a conflict, check whether the `STATUS` field in the output of the [`istioctl authn tls-check`](/docs/reference/commands/istioctl/#istioctl-experimental-authz-check) command
```
Comment (frankbu) [inline content/en/docs/ops/common-problems/network-issues/index.md]: This section of the doc needs to be reworked ... maybe there's a PR in the queue?

File: content/en/news/releases/1.6.x/announcing-1.6/change-notes/index.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
```
Comment (frankbu) [inline content/en/news/releases/1.6.x/announcing-1.6/change-notes/index.md]: We can remove all the 1.6.x boilerplate if we merge @geeknoids code fix: #6783

Comment (ericvn) [reply content/en/news/releases/1.6.x/announcing-1.6/change-notes/index.md]: Latest commit pulls in @geeknoid #6783 and removes all the 1.6.x folders

File: content/en/news/releases/1.3.x/announcing-1.3/change-notes/index.md [hunk 4]
```diff
@@ -43,7 +43,7 @@ aliases:
 
 - **Added** Access Log Service [ALS](https://www.envoyproxy.io/docs/envoy/latest/api-v2/service/accesslog/v2/als.proto#grpc-access-log-service-als) support for Envoy gRPC.
 - **Added** a Grafana dashboard for Citadel monitoring.
-- **Added** [metrics](/docs/reference/commands/sidecar-injector/#metrics) for monitoring the sidecar injector webhook.
+- **Added** metrics for monitoring the sidecar injector webhook.
```
Comment (frankbu) [inline content/en/news/releases/1.3.x/announcing-1.3/change-notes/index.md]:
```suggestion
- **Added** [metrics](https://archive.istio.io/v1.3/docs/reference/commands/sidecar-injector/#metrics) for monitoring the sidecar injector webhook.
```

File: content/en/news/releases/1.4.x/announcing-1.4/change-notes/index.md [hunk 5]
```diff
@@ -47,7 +47,7 @@ weight: 10
 ## `istioctl`
 
 - **Added** the [`istioctl proxy-config logs`](/docs/reference/commands/istioctl/#istioctl-proxy-config-log) sub-command retrieve and update Envoy logging levels.
-- **Updated** the [`istioctl authn tls-check`](/docs/reference/commands/istioctl/#istioctl-authn-tls-check) sub-command to display which policy is in use.
+- **Updated** the [`istioctl authn tls-check`](/docs/reference/commands/istioctl) sub-command to display which policy is in use.
```
Comment (frankbu) [inline content/en/news/releases/1.4.x/announcing-1.4/change-notes/index.md]:
```suggestion
- **Updated** the [`istioctl authn tls-check`](https://archive.istio.io/v1.4/docs/reference/commands/istioctl/#istioctl-authn-tls-check) sub-command to display which policy is in use.
```

File: content/en/docs/ops/common-problems/network-issues/index.md (hunk 2 above)
Comment (ericvn) [reply content/en/docs/ops/common-problems/network-issues/index.md]: Old commands shown on page will still work (they are deprecated). I am just updating the link to point to the new replacement command's ref.

File: content/en/news/releases/1.0.x/announcing-1.0/index.md (hunk 1 above)
Comment (ericvn) [reply content/en/news/releases/1.0.x/announcing-1.0/index.md]: Done

File: content/en/news/releases/1.3.x/announcing-1.3/change-notes/index.md (hunk 4 above)
Comment (ericvn) [reply content/en/news/releases/1.3.x/announcing-1.3/change-notes/index.md]: Done manually

File: content/en/news/releases/1.4.x/announcing-1.4/change-notes/index.md (hunk 5 above)
Comment (ericvn) [reply content/en/news/releases/1.4.x/announcing-1.4/change-notes/index.md]: done manually

File: content/en/docs/ops/common-problems/network-issues/index.md (hunk 2 above)
Comment (frankbu) [reply content/en/docs/ops/common-problems/network-issues/index.md]: Where did you find this replacement? I'm pretty sure there is no replacement for `istioctl authn tld-check`. The following command in the text section won't work, as I understand it.

File: content/en/docs/setup/install/helm/index.md [hunk 6]
```diff
@@ -175,7 +175,7 @@ to manage the lifecycle of Istio.
    If not already defined, create one using following command:
 
     {{< text bash >}}
-    $ kubectl apply -f @install/kubernetes/helm/helm-service-account.yaml@
+    $ kubectl apply -f @manifests/UPDATING-CHARTS.md@
```
Comment (frankbu) [inline content/en/docs/setup/install/helm/index.md]: This doesn't look right ... how can an md file replace a yaml file?

Comment (ericvn) [reply content/en/docs/setup/install/helm/index.md]: There is a follow on PR to update all the links https://github.com/istio/istio.io/pull/6458

**Pull request merged**